api/src/pcapi/routes/backoffice_v3/forms/custom_reimbursement_rule.py

```diff
@@ -31,3 +38,55 @@ def is_empty(self) -> bool:
                 self.offerer.data,
             )
         )
+
+
+class EditCustomReimbursementRule(FlaskForm):
+    offerer = fields.PCTomSelectField(
+        "Structure",
+        multiple=False,
+        choices=[],
+        validate_choice=False,
+        endpoint="backoffice_v3_web.autocomplete_offerers",
+    )
+    subcategories = fields.PCSelectMultipleField(
+        "Sous-catégories", choices=map(lambda s: (s.id, s.pro_label), subcategories_v2.ALL_SUBCATEGORIES)
+    )
+    rate = fields.PCStringField("Taux de remboursement %")
+    start_date = fields.PCDateField("Date de début d'application")
+    end_date = fields.PCDateField(
+        "Date de fin d'application (optionnelle)", validators=(wtforms.validators.Optional(),)
+    )
+
+    def validate_start_date(self, start_date: fields.PCDateField) -> fields.PCDateField:
+        end_date = self._fields["end_date"]
+
+        if end_date.data and (start_date.data > end_date.data):
+            raise wtforms.ValidationError("Ne peut pas être postérieure à la date de fin")
+
+        if start_date.data < (date.today() + timedelta(days=1)):
+            raise wtforms.ValidationError("Ne peut pas commencer avant demain")
+
+        start_date.data = datetime.combine(start_date.data, datetime.min.time())
+        return start_date
+
+    def validate_end_date(self, end_date: fields.PCDateField) -> fields.PCDateField:
+        if end_date.data:
+            end_date.data = datetime.combine(end_date.data, datetime.max.time())
+        return end_date
+
+    def validate_rate(self, rate: fields.PCDateField) -> fields.PCDateField:
```

Comment:
```suggestion
    def validate_rate(self, rate: fields.PCStringField) -> fields.PCStringField:
```
ou `PCDecimalField` si tu essayes ma suggestion d'une réponse précédente